frontend/src/scenes/me/Settings/TwoFactorAuthentication.tsx

```diff
@@ -33,22 +33,19 @@ export function TwoFactorAuthentication(): JSX.Element {
 
             {user?.is_2fa_enabled ? (
                 <>
-                    <div>
-                        <IconCheckmark color="green" />
-                        <b>2FA enabled.</b>
-                    </div>
-                    <br />
-                    <div className="w-60">
-                        <LemonButton type="primary" to="/account/two_factor/" targetBlank={true}>
-                            Manage or disable 2FA
-                        </LemonButton>
+                    <div className="mb-2 flex items-center">
+                        <IconCheckmark color="green" className="text-xl mr-2" />
+                        <span className="font-medium">2FA enabled.</span>
                     </div>
+                    <LemonButton type="primary" to="/account/two_factor/" targetBlank>
+                        Manage or disable 2FA
```

Comment:
This seems like a lot of text for a button, maybe just "Manage"?

Reply:
I agree… I don't love the 2FA settings section at all, so I'd just redo it. But in this PR gonna stick to the existing text.